src/js/providers/bandwidth-monitor.ts

```diff
@@ -1,11 +1,12 @@
 import { BANDWIDTH_ESTIMATE } from 'events/events';
 import { isValidNumber } from 'utils/underscore';
+import { DefaultProvider } from 'types/generic.type';
 
-export default function BandwidthMonitor(provider, initialEstimate) {
-    let bandwidthMonitorInterval = null;
+export default function BandwidthMonitor(provider: DefaultProvider, initialEstimate: number): BandwidthMonitor {
+    let bandwidthMonitorInterval: number | undefined;
     let bandwidthEstimate = initialEstimate;
     return {
-        start() {
+        start(): void {
             this.stop();
             setInterval(() => {
```

Comment:
Is it just me or is `bandwidthMonitorInterval` never being set?

If so this is a bug and it should be set here `bandwidthMonitorInterval = setInterval `